lib/ex_webrtc/rtp/munger.ex

```diff
@@ -0,0 +1,177 @@
+defmodule ExWebRTC.RTP.Munger do
+  @moduledoc """
+  RTP Munger allows for converting RTP packet timestamps and sequence numbers
+  to a common domain.
+
+  This is useful when e.g. changing between Simulcast layers - the sender sends
+  three separate RTP streams (also called layers or encodings), but the receiver can receive only a
+  single RTP stream.
+
+  ```
+  # assume you receive two layers: "high" and "low"
+  # and this is a GenServer
+
+  def init() do
+    {:ok, %{munger: Munger.new(90_000), layer: "high"}}
+  end
+
+  def handle_info({:ex_webrtc, _from, {:rtp, _id, rid, packet}}, state) do
+    if rid == state.layer do
+      {packet, munger} = Munger.munge(state.munger, packet)
+      send_packet_somewhere(packet)
+      {:noreply, %{state | munger: munger}}
+    else
+      {:noreply, state}
+    end
+  end
+
+  def handle_info({:change_layer, layer}, state) do
+    munger = Munger.update(state.munger)
+    {:noreply, %{munger: munger, layer: layer}
+  end
```

Comment:
I belive that in most cases, `rid` is denoted using a single letter so I would keep the convention

```suggestion
  # assume you receive two layers:  "h" ("high") and "l" ("low")
  # and this is a GenServer

  def init() do
    {:ok, %{munger: Munger.new(90_000), layer: "h"}}
  end

  def handle_info({:ex_webrtc, _from, {:rtp, _id, rid, packet}}, state) do
    if rid == state.layer do
      {packet, munger} = Munger.munge(state.munger, packet)
      send_packet_somewhere(packet)
      {:noreply, %{state | munger: munger}}
    else
      {:noreply, state}
    end
  end

  def handle_info({:change_layer, layer}, state) do
    # indicate to the munger that the next packet will be from a new layer
    munger = Munger.update(state.munger)
    {:noreply, %{munger: munger, layer: layer}
  end
```